### DISCUSSION TITLE
wire.model.min.2

### DISCUSSION THREAD

**Discussion by r18s** [livewire/livewire#4734] [3. Feature ideas]
I suggest a new modifier to wire:model.

Just like the **debounce** feature but checking the input length.

**Comment by joshhanley** [comment]
@devmenezes how would you imagine that to work? Not send a request until the minimum characters have been entered? Would it then send a request for every addition character after that?

**Comment by r18s** [comment]
It's just an idea. It might have some side effect that I didn't think
about. I sent it for you to validate. But it would be something like the
attached image.

*<input wire:model.min.2.debounce.750ms="search" />*

**Reply by joshhanley** [reply]
@devmenezes oh I understood your idea 🙂 but I was asking for some further clarification on how you'd imagine it working if it were implemented. Can you answer my questions above? Thanks!

**Comment by r18s** [comment]
*How would you imagine that to work? *As described in the image (last
e-mail).
*Not send a request until the minimum charters have been entered? *Right.
Show the default list or value in this case.
*Would it then send a request for every addition character after that? *Yes,
respecting the debounce if it is set.

Em seg., 28 de fev. de 2022 às 17:30, Josh ***@***.***>
escreveu:

**Reply by joshhanley** [reply]
@devmenezes yep that all makes sense. Yeah would be good if it respects the debounce too. PR's are welcome!
